experiment_prototype/scan_classes/sequences.py

```diff
@@ -337,107 +260,97 @@ def __init__(self, seqn_keys, sequence_slice_dict, sequence_interface, transmit_
         self.numberofreceivesamples = int(self.transmit_metadata['rx_sample_rate'] * self.sstime *
                                           1e-6)
 
-        self.first_rx_sample_time = 0  # initilized only but set in build_pulse_transmit_data
-        self.blanks = []
+        sample_time = (int(self.combined_pulses_metadata[0]['total_pulse_len'] / 2) + tr_window_time) / txrate
```

Comment:
Also, it should be based on the first pulse of the first slice's number of samples, to align for the first slice I think

Reply:
Well given that we can use sequence offsetting, shouldnt this be a per slice number?

Reply:
I don't think it can be a per slice thing, this is just for setting the alignment of the decimator so that it lines up with the middle of the first pulse (whichever slice has the first pulse). The rest of the samples will just be placed according to the output sample rate. If the rest of the pulses don't align - that's up to the experimenter